app/core/Router.coffee

```diff
@@ -13,7 +13,24 @@ module.exports = class CocoRouter extends Backbone.Router
     @initializeSocialMediaServices = _.once @initializeSocialMediaServices
 
   routes:
-    '': go('HomeView')
+    '': ->
+      # Testing new home page
```

Comment:
This should be in User.coffee with the rest of the a/b tests, so they're all in one place.  `getHomepageGroup()`